config.rb

```diff
@@ -57,3 +57,25 @@ def sanitize(contents)
     proxy resource[:path], resource[:template], resource[:frontmatter]
   end
 end
+
+# Configuration for Analytics pages
+ignore "analytics/templates/*"
+
+page "analytics/*", layout: :analytics_layout
+
+# rubocop:disable Style/HashSyntax
```

Comment:
Why do we need this comment?

Reply:
Ah, that sets the layout for all files under `/analytics` - I feel this is better, safer and easier than having to add the `layout` metadata to each page. 

I'll add a comment to the commit to make it clear why I'm adding this.

Reply:
Sorry, I meant "why do we need rubocop:disable Style/HashSyntax?"

Reply:
Oh, sorry! I don't believe Middleman is Ruby 3.2 friendly, so unsure (if removed and Rubocop fixed up) this will work - I'll give it a try...

Reply:
Nope you're right we don't need this. Great spot - thanks @richardTowers 👍